core/src/main/java/com/sequenceiq/cloudbreak/reactor/CoreVerticalScaleHandler.java

```diff
@@ -37,6 +39,9 @@ public class CoreVerticalScaleHandler implements CloudPlatformEventHandler<CoreV
     @Inject
     private StackUpscaleService stackUpscaleService;
 
+    @Inject
+    private AwsResourceService awsResourceService;
```

Comment:
This class (CoreVerticalScaleHandler) should not be referencing anything that is cloud provider specific. Whatever AwsResoruceService does needs to be abstracted to apply across the cloud providers.

Reply:
done